docs/UserGuide.md

```diff
@@ -382,7 +382,8 @@ The simplest way to analyze multiple repos in one go is to use the `--repos` par
 
 Alternatively, you can use csv config files to customize the analysis as before while specifying multiple repos to analyze.
```

Comment:
Can be updated to simply
`Alternatively, you can use csv config files to further customize the analysis `